### PR TITLE
Fix #4782 kickmap error

### LINKED ISSUE #4782: kickmap sdds report broken
The functions refer to outdated model schema, similar to #4780

### PR DESCRIPTION
**Pull request by mkeilman**
This also cleans up `_save_kick_map_sdds()` and removes the download as text option, which was never implemented and is of unclear importance.

### CODE REVIEW

**Comment by mkeilman** [comment]
Kickmaps are for undulator magnets, so this can be tested with the one under Examples.

File: sirepo/template/radia.py [hunk 1]
```diff
@@ -1233,29 +1231,15 @@ def _save_fm_sdds(name, vectors, scipy_rotation, path):
...
-    for i in range(len(x_vals)):
-        for j in range(len(y_vals)):
-            h.append(h_vals[i][j])
-            v.append(v_vals[i][j])
-    col_data.append([x])
-    col_data.append([y])
-    col_data.append([h])
-    col_data.append([v])
+    col_data = [
+        [numpy.tile(0.001 * numpy.array(km_data.x), len(km_data.x)).tolist()],
```
Comment (gurhar1133) [inline sirepo/template/radia.py:1238]: maybe put 0.001 in a constant with a name that clarifies it's role?

**Review by gurhar1133 — APPROVED** [review]
I approved with one suggestion, see comments.

But looks good

File: sirepo/template/radia.py (hunk 1 above)
Comment (mkeilman) [reply sirepo/template/radia.py]: Good call, [86bc328](https://github.com/radiasoft/sirepo/pull/4785/commits/86bc3289d0fd30f6855bef62135ad38a493cf383)

**Pull request merged**